walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -130,6 +132,80 @@ public void everyAttendeeIsConsidered() {
     Assert.assertEquals(expected, actual);
   }
 
+  @Test
+  public void everyAttendeeIsConsideredWithInvalOptional() {
+    // Have each person have different events. We should see two options because each person has
```

Comment:
Looks like your lines are over the character limit here. Would you mind wrapping them? The same goes for some other comments in this file

Reply:
The auto formatter didn't have issue with this specific comment. I run it and it cleaned some of the comments up but not this one. Could there be an issue with my formatter?

Reply:
Hmm I wonder if there's a setting where you can set the maximum line limit? It's also possible the formatter doesn't try to modify comments at all: I'm not sure all of them do so, so you may have to manually make sure comments aren't over the line limit.